source-academy/cadet-frontend
# Fix components extending beyond screen dimensions on mobile view

### Description



Closes #2241 

The following components extend beyond the screen dimensions on mobile view:
- `ConfirmDialog` and `PromptDialog`
- SICP "Text Search" and "Index Search" search bar
- Assessment `OverviewCard`

### Type of change


### How to test



On a Mobile view:

`ConfirmDialog` and `PromptDialog`
1. In Playground, switch to Source 2, 3 or 4 to access and "Enable Folder mode"
2. Create a "New File" or "New Directory" with a long name.
3. Try to "Delete" the file and the Dialog box will appear.

SICP "Text Search" and "Index Search" search bar
1. In SICP JS, click "Text Search" or "Index Search" to display the search bar.

Assessment `OverviewCard`
1. In the Assessments tabs (e.g. Missions, Quests), the Assessment OverviewCard layout should display with the icon on top of the description.

### Checklist
